tests/unit/test_units.py

```diff
@@ -42,8 +49,6 @@ def test_nox():
         np.testing.assert_allclose(N.to("NOx").magnitude, 46 / 14)
         np.testing.assert_allclose(NO2.to("NOx").magnitude, 1)
         np.testing.assert_allclose(NOx.to("NO2").magnitude, 1)
-        # this also becomes allowed, unfortunately...
-        np.testing.assert_allclose(NOx.to("N2O").magnitude, 44 / 46)
```

Comment:
Should this be added as a test that it explicitly doesn't work anymore?

Reply:
very nice, added